pythran/config.py

```diff
@@ -1,8 +1,9 @@
+from __future__ import unicode_literals
 try:
     # python3 vs. python2
-    import configparser
+    from configparser import ConfigParser
 except ImportError:
-    import ConfigParser as configparser
+    from ConfigParser import SafeConfigParser as ConfigParser
```

Comment:
why this change?

Reply:
To avoid:
```python
>>> cfg = SafeConfigParser().read(["~/.pythranrc"])
__main__:1: DeprecationWarning: The SafeConfigParser class has been renamed to ConfigParser in Python 3.2. This alias will be removed in future versions. Use ConfigParser directly instead.
```

Reply:
cool, thanks!